### ISSUE TITLE
Mention the DAAC bucket access example algorithm in docs

### ISSUE THREAD

**Issue by emileten** [MAAP-Project/maap-documentation#355]
I just finished setting up an example repository with a DPS algorithm definition where we show how to access DAAC buckets in DPS. It's well documented with examples, etc. 

It would be great to put this somewhere in the docs. Maybe it could go in the soon-to-be-updated DPS documentation page @rtapella @wildintellect ? 

Link to the repository : https://github.com/MAAP-Project/example-dps-daac-access

**Comment by wildintellect** [comment]
@emileten I think it should go in https://docs.maap-project.org/en/latest/technical_tutorials/accessing.html
@jjfrench where did your ADE example end up?

**Comment by rtapella** [comment]
I agree with @wildintellect … I think it should go in a data-access section. Maybe in the DPS tutorial we should have some examples of accessing data from various sources, from inside an algorithm. For the initial version we are keeping it simpler.
